common/config/babel/features.js

```diff
@@ -57,8 +57,6 @@ module.exports = {
     "unsupported-browser",
     // Feature to show the total number of participants in a call (currently in beta in calling SDK, hence this must be conditionally compiled)
     "total-participant-count",
-    // feature for tracking environment info API different than unsupported browser. stable use of environment info affects other components possibly sooner
```

Comment:
This should've been removed as part of #5328 , couple lines remained that was related to this PR work so quickly removed